ginmetrics/middleware.go

```diff
@@ -33,6 +33,19 @@ func (m *Monitor) Use(r gin.IRoutes) {
 	})
 }
 
+// Pass endpoints to monitor
```

Comment:
Can the comments be modified in more detail here?

such as:

// UseWithoutExposingEndpoint used to add monitor interceptor to gin router.
// But there is not set the http path, users can use Expose function to set it.
// Because some users need to expose metrics to other port.